samples/java-eventsourced-shopping-cart/src/main/proto/com/example/shoppingcart/view/shopping_cart_view_model.proto

```diff
@@ -19,11 +19,12 @@ package com.example.shoppingcart.view;
 option java_outer_classname = "ShoppingCartViewModel";
 
 import "akkaserverless/annotations.proto";
+import "akkaserverless/codegen/annotations.proto";
 import "com/example/shoppingcart/domain/shoppingcart_domain.proto";
 
 service ShoppingCartViewService {
-  option (akkaserverless.service) = {
-    type : SERVICE_TYPE_VIEW
+  option (akkaserverless.codegen) = {
```

Comment:
Nice that it is explicit that it is for codegen now, I was honestly not 100% sure before if any parts were read and handed to the proxy.